docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1619,359 +1635,597 @@ We **strongly recommend** using these markers any time when you use an extended
+
+**Syntax**
+
+```
+[target]:is(selectors)
+```
+- `target` — optional, standard or extended css selector, can be missed for checking *any* element
+- `selectors` — [*forgiving selector list*](https://drafts.csswg.org/selectors-4/#typedef-forgiving-selector-list) of standard or extended selectors. For extended selectors only compound selectors are supported, not complex.
+
+##### `:is()` limitations {#extended-css-is-limitations}
+
+> Rules with the `:is()` pseudo-class should use the [native implementation of `:is()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) if rules use `##` marker and it is possible, i.e. with no other extended selectors inside. To force applying ExtendedCss rules with `:is()`, use `#?#`/`#$?#` marker explicitly.
+
+> If the `:is()` pseudo-class argument `selectors` is an extended selectors, due to the way how the `:is()` pseudo-class is implemented in ExtendedCss v2.0, it is impossible to apply it to the top DOM node which is `html`, i.e. `#?#html:is(<extended-selectors>)` does not work. So if `target` is not defined or defined as a [universal selector](https://www.w3.org/TR/selectors-4/#the-universal-selector) `*`, the extended pseudo-class applying is limited to **`html`'s children**, e.g. rules `#?#:is(...)` and `#?#*:is(...)` are parsed as `#?#html *:is(...)`. Please note that there is no such limitation for a standard selector arg, i.e. `#?#html:is(.locked)` works fine.
+
+> [Complex selectors](https://www.w3.org/TR/selectors-4/#complex) with extended pseudo-classes are not supported as `selectors` argument for `:is()` pseudo-class, only [compound ones](https://www.w3.org/TR/selectors-4/#compound) are allowed. It is [one of known issues](#known-issues). Check examples below for more details.
+
+**Examples**
+
+`#container *:is(.inner, .footer)` selects only the element `div#target1`:
+```html
+<!-- HTML code -->
+<div id="container">
+  <div data="true">
+    <div>
+      <div id="target1" class="inner"></div>
+    </div>
+  </div>
+</div>
+```
+
+Due to limitations `:is(*:not([class]) > .banner)'` does not work
+but `:is(*:not([class]):has(> .banner))` can be used instead of it to select the element `div#target2`:
+```html
+<!-- HTML code -->
+<span class="span">text</span>
+<div id="target2">
+  <p class="banner">inner paragraph</p>
+</div>
+```
+
+
+#### Pseudo-class `:not()` {#extended-css-not}
+
+The `:not()` pseudo-class allows to select elements which are *not matched* by selectors passed as arg. Invalid argument selectors are not allowed and error is to be thrown. Our implementation of the [`:not()` pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:not).
+
+**Syntax**
+
+```
+[target]:not(selectors)
+```
+- `target` — optional, standard or extended css selector, can be missed for checking *any* element
+- `selectors` — selector list of standard or extended selectors
```

Comment:
selector list of ... -> list of ...

I believe the word 'selector' doesn't add any meaning in this case